authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerHelper.java

```diff
@@ -86,6 +86,9 @@ public RangerHelper(
    */
   void checkPolicyItemAccess(RangerPolicy.RangerPolicyItem policyItem)
       throws AuthorizationPluginException {
+    if (!isGravitinoManagedPolicyItemAccess(policyItem)) {
+      return;
```

Comment:
Emm...
Not sure I understand the invocation protocol here.
From the function comment, I can guess that the function (a check) succeeds when it returns, and we use an exception to signify a failure.

If that is the case, should we raise an exception on line 90, rather than pretending it was a success?

Reply:
You are right. I modified the function but did not change the comment, so it caused confusion.

There are two types of policy items:

   * Gravitino managed policy items: They contain one privilege each.
   * User-defined policy items: They could contain multiple privileges and not be managed and checked by Gravitino.

I have fixed the comment now.

Reply:
line 91: this function only checks managed policy, that is fine.
line 94: there should be only one item in the managed policy, this is fine too.
line 98 and then on: check if there are duplicate items ... how could that happen?

Reply:
> line 91: this function only checks managed policy, that is fine. line 94: there should be only one item in the managed policy, this is fine too. line 98 and then on: check if there are duplicate items ... how could that happen?

fixed it